x-pack/plugins/apm/public/plugin.ts

```diff
@@ -346,6 +347,17 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
         view: 'package-policy-edit',
         Component: getLazyAPMPolicyEditExtension(core),
       });
+
+      fleet.registerExtension({
+        package: 'apm',
+        view: 'package-policy-edit-tabs',
+        tabs: [
+          {
+            title: 'APM Agents',
+            Component: getLazyAPMAgentConfigEditExtension(core),
+          },
```

Comment:
I was expecting that the `Settings` tab would be defined here too. Am I wrong?

Reply:
I intentionally make this extension point only for _additional_ tabs, with the first 'Settings' tab still being controlled by he `package-policy-edit` extension point view. This was so there was no breaking change in the extensions API and to also avoid conflicts / collisions between the main settings and tabs. If the app doesn't use the `package-policy-edit-tabs` then there is no change needed.

Reply:
Does `package-policy-edit-tabs` depend on the existence of `package-policy-edit`?

Reply:
no, if `package-policy-edit` is not used, then it would just show the out-of-the-box settings form generated by default from the package definition in the first Tab.